plugins/lua/spectate.lua

```diff
@@ -84,7 +84,8 @@ local function load_state()
            function() config_file:write() end
 end
 
-local config, save_state = load_state()
+-- global variables to make the UI script simpler
+_ENV.config, _ENV.save_state = load_state()
```

Comment:
No need for the _ENV prefix. Just non local is equivalent.

Reply:
I know, but this way I'm signaling this is intended as a public variable.
I'd also get a "Global variable in lowercase initial" diagnostic:

![image](https://github.com/user-attachments/assets/f1e82b4f-dd01-4078-adfc-49586e56b536)

(which could be resolved by renaming, but the signaling issue remains)

Reply:
(and using `local M = {}` for the module table would make public declarations just two characters - `M.` - instead of 5 - `_ENV.`)

Reply:
(but that would depend on the exact version of Lua and what `mkmodule` does, because we definitely do want to have a local environment table)

Reply:
Alternately, could add public functions that return the local vars, if that's cleaner